pandas/tests/series/test_timeseries.py

```diff
@@ -1100,3 +1100,15 @@ def test_asarray_tz_aware(self):
             result = np.asarray(ser, dtype=object)
 
         tm.assert_numpy_array_equal(result, expected)
+
+    def test_NA_values_with_cache(self, unique_nulls_fixture,
```

Comment:
this needs to be in pandas/tests/indexes/datetimes/test_tools

Reply:
put near other tests for cache

Reply:
Done